### ISSUE TITLE
Warnings with Java 9

### ISSUE THREAD

**Issue by sachin-walia** [networknt/light-4j#103]
I launched the app with newly released JDK 9 and following warnings are printed:

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.xnio.nio.NioXnio$2 (file:/app/target/myapp-1.4.0.jar) to constructor sun.nio.ch.KQueueSelectorProvider()
WARNING: Please consider reporting this to the maintainers of org.xnio.nio.NioXnio$2
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by stevehu** [comment]
@sachinwalia2k8 Thanks for testing on Java 9. I haven't got a chance to try it although have a roadmap to move it Java 9 once it is stable. There are a lot of features in Java 9 that we can leverage. The particular warning is for Xnio which is used by Undertow for low level IO over the socket. I am pretty sure that someone has opened a ticket to get it addressed in Xnio project and I will double check it. As the warning message states, it still works for the current version of Java 9 but the future is not guaranteed. It will take some time for all the dependent project to move to Java 9 I think. We will be moving to it once the underline dependencies are ready. Thanks for bringing it up and let's keep this issue open for tracking purpose.

**Comment by sachinwalia2k8** [comment]
@stevehu wondering if there is any progress on Java 9 compatibility. Moreover undertow team has updated their dependencies to support Java 9. I think undertow version 2.0 will be Java 9 compatible.

**Comment by stevehu** [comment]
Undertow 2.0 is just in beta and we can upgrade to it once it is stable. Are you planning to use Java 9 on production?

**Comment by sachinwalia2k8** [comment]
I am right now on Java 8 but very interested in upgrading to Java 9. However until there is stability on Java 9 may not be able to do so.

**Comment by stevehu** [comment]
As we have some client running some of the framework at scale, I cannot move to Java 9 in the main release but maybe 2.x release in separate branch. These companies will only move to Java 11 which is long term support once that version is stable.

**Comment by stevehu** [comment]
We have released the 2.0.0 beta with JDK 11. Closing this issue.
